src/MySqlConnector/Core/ConnectionPool.cs

```diff
@@ -67,17 +67,9 @@ public async ValueTask<ServerSession> GetSessionAsync(MySqlConnection connection
 				else
 				{
 					if (ConnectionSettings.ConnectionReset || session.DatabaseOverride is not null)
-					{
```

Comment:
The release notes are confusing. It's not clear that an unstable pooled connection may only be returned if `ConnectionReset = false`, can worry anyone who did not dive into the code to look at this set of IFs. Please update the release notes :-)

Reply:
> If Connection Reset = False, MySqlConnection.Open will return a pooled connection (if one is available) without testing it for liveness (by pinging the server). 

Do you want this to say "If and only if"? Do you have a suggested edit?

Reply:
Just make it clearer for which "if" this paragraph belongs too :)

When ConnectionReset=false, if the server has closed the pooled connection, MySqlConnection.Open will succeed, but the first operation on that connection will fail. Command execution should be wrapped in a try/catch/retry loop to handle this scenario.

Reply:
Ah, gotcha. Yes, I see now that that is confusing. (Should have kept reading 😀)

Reply:
Consolidated the bullet points here: https://github.com/mysql-net/MySqlConnector/commit/5151ca4df84103fc954c25a699bb8881d5a5ab99